src/app/pages/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,13 @@
+<app-hill-background cover></app-hill-background>
+<app-back-button top-left routerLink="/"></app-back-button>
+<div class="centered-container">
+  <div>
+    <mat-card>
+      <mat-card-title class="title">Whoops</mat-card-title>
+      <mat-card-content>
+        This page doesn't seem to exist.
+      </mat-card-content>
+    </mat-card>
+  </div>
+</div>
```

Comment:
I think we should try to have all our errors be consistent so this would change it to be more like the invalid round error and others:
```suggestion
<app-hill-background cover></app-hill-background>
<div class="centered-container">
  <div>
    <mat-card>
      <mat-card-title>Whoops</mat-card-title>
      <mat-card-content>
        This page doesn't seem to exist.
      </mat-card-content>
      <mat-card-actions align="end">
        <button mat-button color="primary" routerLink="/">Go Back</button>
      </mat-card-actions>
    </mat-card>
  </div>
</div>
```

We might want to say "Go Home" rather than "Go Back" but I'm not completely sure.